template/include/_config.hpp

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
This should be `#pragma once`'d and should be included first in most files as a matter of convention